internal/appsec/dyngo/instrumentation/grpcsec/grpc.go

```diff
@@ -39,6 +41,8 @@ type (
 		dyngo.Operation
 		instrumentation.TagsHolder
 		instrumentation.SecurityEventsHolder
+		UnaryHandler  grpc.UnaryHandler
+		StreamHandler grpc.StreamHandler
```

Comment:
what about making it super simple for now by simply having a `blocked code.Code` field here in place of those two?
We know the current set of "more dynamic action parameters" don't apply to gRPC for now.

Reply:
Took a stab at it in 9f6411eafd0a2